.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "carla_msgs"]
 	path = carla_msgs
-	url = https://github.com/carla-simulator/ros-carla-msgs
-	branch = master
+	url = https://github.com/Motor-Ai/carla-simulator-ros-carla-msgs
+	branch = ue4-extend-ros2
```

Comment:
This branch name has to be changed to the dev branch of the carla-msgs after the merge of the commit into that branch.
In each case a new branch is required also for the ros-bridge itself ...